pkg/kn/commands/service/create_mock_test.go

```diff
@@ -681,8 +681,12 @@ func TestServiceCreateWithAutoScaleServiceAnnotationsError(t *testing.T) {
 
 func getServiceWithUrl(name string, urlName string) *servingv1.Service {
 	service := servingv1.Service{}
-	url, _ := apis.ParseURL(urlName)
-	service.Status.URL = url
 	service.Name = name
+	setUrl(&service, urlName)
 	return &service
 }
+
+func setUrl(service *servingv1.Service, urlName string) {
+	url, _ := apis.ParseURL(urlName)
```

Comment:
why are ignoring errors here? the parse could fail, no?

Reply:
this is just extracted from create test and this is a test we have full control of the input as well so I think it perfectly valid in this case to omit error handling when don't use this method with an invalid parameter (as mentioned, this is our own test code not exposed for public usage)